### PR TITLE
[feat] Join/Leave event

### PR DESCRIPTION
**Pull request by rado196**
This PR includes the `join event` and `leave event` functionality on the mobile app.

### CODE REVIEW

File: clients/app/src/views/screens/Event/components/EventComponent.tsx [hunk 1]
```diff
@@ -21,12 +21,41 @@ import Icon from 'views/components/base/Icon';
...
 import { getReadableNumber } from 'utils/helpers/readableNumber';
 import { IEventComponentProps } from '../types';
+import buildEventMembersStore from 'stores/eventMembers';
 
 const EventComponent: React.FC<IEventComponentProps> = ({ data }) => {
   const { pushTo } = useNavigationWithParams();
   const { colors } = useTheme();
   const [ratio, setRatio] = React.useState(3 / 2);
 
+  const eventMembersStore = React.useMemo(() => buildEventMembersStore(), []);
```
Comment (hay92k) [inline clients/app/src/views/screens/Event/components/EventComponent.tsx:31]: Is not it better to move outside of Component?

File: clients/app/src/views/screens/Event/components/EventComponent.tsx [hunk 2]
```diff
@@ -21,12 +21,41 @@ import Icon from 'views/components/base/Icon';
...
+
+    if (isJoined) {
+      eventMembersStore.remove({ eventId: data.id });
+    } else {
+      eventMembersStore.setFieldValue('eventId', data.id);
+      eventMembersStore.create();
+    }
+  }, [data.id, isJoined, eventMembersStore]);
+
+  React.useEffect(() => {
```
Comment (hay92k) [inline clients/app/src/views/screens/Event/components/EventComponent.tsx:50]: Isn't it better to write this part like this?

```
if (storeAdd.status === 'error')
  setIsJoined(!isJoined);
```

**Review by hay92k — CHANGES_REQUESTED** [review]

File: clients/app/src/views/screens/Event/components/EventComponent.tsx (hunk 1 above)
Comment (rado196) [reply clients/app/src/views/screens/Event/components/EventComponent.tsx:31]: same in PR: https://github.com/ultras-io/ultras/pull/228

File: clients/app/src/views/screens/Event/components/EventComponent.tsx (hunk 2 above)
Comment (rado196) [reply clients/app/src/views/screens/Event/components/EventComponent.tsx:50]: same in PR: https://github.com/ultras-io/ultras/pull/228

**Review by hay92k — APPROVED** [review]
lgtm

**Pull request merged**